### PR TITLE
Minor fixup for #8293

### PR DESCRIPTION
**Pull request by permcody**
Just tweaking the changes Andy made to modules.mk for him. This supersedes #8293.

### CODE REVIEW

**Comment by permcody** [comment]
Looks like Github won't let me review my own PR (even though it's almost all Andy's). Assigning to @dschwen

**Comment by WilkAndy** [comment]
Aha, thank you for fixing the makefile stuff, @permcody .  Hopefully next time i'll be able to do it on my own.

a

File: unit/Makefile [hunk 1]
```diff
@@ -25,12 +25,13 @@ include $(FRAMEWORK_DIR)/build.mk
...
-XFEM              := yes
-POROUS_FLOW       := yes
-FLUID_PROPERTIES  := yes
+SOLID_MECHANICS    := yes
+TENSOR_MECHANICS   := yes
+PHASE_FIELD        := yes
+XFEM               := yes
+POROUS_FLOW        := yes
+FLUID_PROPERTIES   := yes
+CHEMICAL_REACIONTS := yes
```
Comment (jwpeterson) [inline unit/Makefile]: Typo on this line!  "REACIONTS"

**Review by jwpeterson — CHANGES_REQUESTED** [review]

File: unit/Makefile (hunk 1 above)
Comment (permcody) [inline unit/Makefile]: Nice catch!

File: modules/porous_flow/include/kernels/PorousFlowDesorpedMassTimeDerivative.h [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef PORFLOWDESORPEDMASSTIMEDERIVATIVE_H
```
Comment (dschwen) [inline modules/porous_flow/include/kernels/PorousFlowDesorpedMassTimeDerivative.h]: why not `POROUSFLOWDESORPEDMASSTIMEDERIVATIVE_H` like in all the other files?

Comment (WilkAndy) [inline modules/porous_flow/include/kernels/PorousFlowDesorpedMassTimeDerivative.h]: Hmmm.  i'll change this

**Comment by WilkAndy** [comment]
@permcody , i think i just made a PR to your wilk_pr2 branch that changes PORFLOW to POROUSFLOW in 2 header files (to address @dschwen comment above).  Could you tell me if i've done the wrong thing?

**Comment by WilkAndy** [comment]
Oh, you're too fast for me @permcody , thanks.

**Comment by permcody** [comment]
@jwpeterson - perhaps you can review this PR since I'm now an author. It's been up here for awhile now.

**Review by jwpeterson — APPROVED** [review]

**Pull request merged**